File: PHP/Program/Device.php
<?php
require_once 'Electronic.php';

class Device extends Electronic {
    private $brand = '';
    private $warranty = 0;
    private $power = 0;

    public function __construct($id = 0, $nama = '', $price = 0, $stock = 0, $photo = '', $brand = '', $warranty = 0, $power = 0) {
        parent::__construct($id, $nama, $price, $stock, $photo);
        $this->brand = $brand;
        $this->warranty = $warranty;
        $this->power = $power;
    }

    public function setBrand ($brand) { $this->brand = $brand; }
    public function getBrand () { return $this->brand; }

    public function setWarranty ($warranty) { $this->warranty = $warranty; }
    public function getWarranty () { return $this->warranty; }

    public function setPower ($power) { $this->power = $power; }
    public function getPower () { return $this->power; }
}
?>

File: Python/Program/Device.py
from Electronic import Electronic

class Device (Electronic):
    __brand = ""
    __warranty = 0
    __power = 0

    def __init__(self, id = 0, name = "", price = 0, stock = 0, brand = "", warranty = 0, power = 0):
        super().__init__(id, name, price, stock)
        self.__brand = brand
        self.__warranty = warranty
        self.__power = power

    def setBrand(self, brand):
        self.__brand = brand

    def getBrand(self):
        return self.__brand

    def setWarranty(self, warranty):
        self.__warranty = warranty

    def getWarranty(self):
        return self.__warranty

    def setPower(self, power):
        self.__power = power

    def getPower(self):
        return self.__power